test/image/mocks/gl3d_opacity-surface.json

```diff
@@ -1540,14 +1540,23 @@
     "range": [1, 2],
     "gridcolor": "rgb(255, 255, 255)"
    },
-   "cameraposition": [
-    [
-     -0.8326258918343652, -0.31013201281074904, -0.14770225080852764,
-     0.4344379172943774
-    ],
-    [-0.09586835824120499, -0.07248137902934104, 0.013764797418843955],
-    1.9263438581634802
-   ],
+   "camera": {
+    "eye": {
+     "x": 0.8970225259545955,
```

Comment:
aaaand we just *know* these numbers? :-)

Reply:
I used [the existing conversion function](https://github.com/plotly/plotly.js/pull/7217/files#diff-39ff2a6111d28f1f7183c2927e07895b5adb4f977352fb762040d63628af4a68) before deleting it. :-) 

But I'll admit I don't actually know what the numbers in the old format mean. The meaning of the new format is documented [here](https://plotly.com/javascript/reference/#layout-scene-camera).